### PR TITLE
Feat/backend

### PR DESCRIPTION
**Pull request by ilyasfoo**

### CODE REVIEW

**Comment by ilyasfoo** [comment]
Please check the output data at: https://github.com/Code-for-All/lockdown/pull/65 and https://github.com/Code-for-All/lockdown/pull/64 and let me know if the structure is desired

File: backend/src/loaders/territory/territory.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export function parseData(territory) {
+  return {
+    contribute_url: `${lockdownSheetUrl}/edit#gid=${territory['googlesheet_tab_id']}`,
+  }
+}
+
+export async function getData() {
+  const territories = {};
+  const territoriesMeta = require('./meta.json');
+  each(territoriesMeta, (territory, isoCode) => {
```
Comment (thepassle) [inline backend/src/loaders/territory/territory.js]: doesnt this do the same as Array.forEach?

Comment (ilyasfoo) [reply backend/src/loaders/territory/territory.js]: Actually it was an object, not array. Anyways I've replaced with native implementation, probably going to ditch lodash soon if no use case

File: backend/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+## Backend scripts to load and transform data
+
+### Requirements:
+Minimum node version: 8.5
+
+#### Installation:
+```
+npm install
+```
+You will need a credentials.json generated from the google developer for sheets api v4 in the base folder
```
Comment (LarsDenBakker) [inline backend/README.md]: Can you explain how this works a bit more? What are the steps to take?

**Comment by LarsDenBakker** [comment]
This looks great! As discussed on slack, if we can add the option to add tokens through environment variables we can run the scripts in the CI as well.

File: backend/README.md (hunk 2 above)
Comment (ilyasfoo) [reply backend/README.md]: For CI, I think its simply:
- Set the env variables
- Run `npm install`
- Run `npm start`

Then it should spew out some messages as it goes along

**Comment by ilyasfoo** [comment]
> This looks great! As discussed on slack, if we can add the option to add tokens through environment variables we can run the scripts in the CI as well.

Yes, now you can use `GOOGLE_SERVICE_ACCOUNT_EMAIL` and `GOOGLE_PRIVATE_KEY`.
These can be obtained from the generated `credentials.json`

**Pull request merged**